Data-Analytics/Spark/README.md

```diff
@@ -18,8 +12,8 @@ of delivering app's **.jar** file.
 
 | Parameter               | Description                                                                                            | Example Value                                                                                                                                                                                                                                                                                                                                          |
 |-------------------------|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `*.image`               | Image that is used to run Spark itself                                                                 | <ul><li>`gcr.io/mapr-252711/ezaf-spark-demo-example:fy23-q3-fts`</li>Custom image with `.jar` file embedded<br/><br/><li>`gcr.io/mapr-252711/spark-3.4.0:v3.4.0`</li>Pure Spark image<ul>                                                                                                                                                              |
- | `*.mainApplicationFile` | Path inside the pod that is used by spark to find `.jar` file with app                                 | <ul><li>`local:///tmp/DataProcessTransfer.jar`</li>Path in which `.jar` file was embedded into image<br/><br/><li>`local///mounts/data/DataProcessTransfer.jar`</li>:warning: The specified path must be mounted to the `PV`<br/><br/><li>`maprfs:///home/spark/DataProcessTransfer.jar`</li>Path in which `.jar` file can be reached im `MaprFS`</ul> |
+| `*.image`               | Image that is used to run Spark itself                                                                 | <ul><li>`gcr.io/mapr-252711/spark-3.5.0:v3.5.0`</li>Pure Spark image<ul>                                                                                                                                                              |
+ | `*.mainApplicationFile` | Path inside the pod that is used by spark to find `executable` file with app                                 | <ul><li>`local:///tmp/<Application>.jar`</li>Path in which `.jar` file was embedded into image<br/><br/><li>`local///mounts/<Application>.jar`</li>:warning: The specified path must be mounted to the `PV`<br/><br/><li>`maprfs:///home/spark/<Application>.jar`</li>Path in which `.jar` file can be reached im `MaprFS`</ul> |
```

Comment:
Technically its .jar file, not an executable, like ELF or script with execute permission.